### ISSUE TITLE
Password Reset does not work

### ISSUE THREAD

**Issue by konwin** [nextcloud/docker#814]
Hello and first sorry for this double post, but I got the feeling that my comment under the other issue were not getting any attention.

I have the following issue:
When any existing user wants to reset his password or a new user without a password is asked to "reset" his passord the site for password reset loads and I can enter a password but after entering a password and pressing the button nothing happens!
Firefox tells me the password field is "insecure", but the connection details show my certificate and that the site is secure!
Other browser do not work either
My nextcloud installation is a docker one running behind a traefik proxy with a wildcard certificate.
Here is my [docker-compose.yml](https://github.com/nextcloud/docker/files/3372170/docker-compose.yml.txt)
Any ideas how to track whats going wrong?

**Comment by DatAres37** [comment]
Did you properly setup your config.php by adding the necessary reverse proxy information?
Can you find any http-URLs in the source code of the page?

**Comment by konwin** [comment]
Ok, found some time to sort that out.
I inspected the set password page (I created a new user without a password and used the link from the email to set my password) in Firefox and actually the link for password reset button was only http
`<form action="http://nextcloud...../Username" id="reset-password" method="post">`
I changed it "on-the-fly" with the Inspector to https and was able to push the button and the password was set and active!
So I looked up the config.php settings you are mentioning. Until now I only added
`  'trusted_proxies' =>
  array (
    'myreverseproxysIP',
  ),`
additionally I added
`  'overwriteprotocol' => 'https',`
and after a restart of nextcloud I got a working password reset page!
So thank you for pointing me to the setup of config.php and sorry for not reading the entire manual.
